docs/pull_request_template.md

```diff
@@ -0,0 +1,5 @@
+## Maintainers will complete the following section
+
+- [ ] Commit messages are descriptive enough ([hints](https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/))
+- [ ] Code coverage from testing does not decrease and new code is covered
+- [ ] [Controllers diagrams](https://github.com/redhat-appstudio/integration-service/tree/main/docs) are updated when PR changes controllers code  (if applicable)
```

Comment:
Do we also need a line to remind us to test our code locally in a cluster (if applicable)?

Reply:
That should be very automatic.

This is also maintainers checklist, not dev checklist.
Do you want me to add dev checklist part there?

Reply:
I think that the local testing part can stay as part of the definition of done document.

Reply:
can you please remind me of the difference between maintainers checklist and dev checklist?

Reply:
Oh I see, then it makes sense to not include the local testing thing here because that should stay in the definition of done doc. Personally, if the checklist is really long I end up ignoring it, so I'm good with this.